config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -867,6 +867,7 @@ def generate_misc():
                    extra_flags=[
                        "--image=cos-cloud/cos-105-17412-156-49",
                        "--set=spec.nodeProblemDetector.enabled=true",
+                       "--gce-service-account=default",
```

Comment:
As part of the scale testing changes to `kubetest2 kops`, this flag needs to be set manually on all GCE jobs

https://github.com/kubernetes/kops/pull/16181/files#diff-93854b73be94e9c045d6ad214ead9b77528a510ef1d19285c8699c9d6697b410L195